cmd/ipfs/daemon.go

```diff
@@ -457,6 +469,99 @@ func daemonFunc(req *cmds.Request, re cmds.ResponseEmitter, env cmds.Environment
 	return errs
 }
 
+const mfsRepinInterval = time.Minute
```

Comment:
@petar @lidel is this the kind of thing users are going to want to change from a config file? They can always pin more manually if they're really concerned, but was wondering aloud here. If this could get more complicated we may want further nesting in the config like `Policies.PinMFS.Enabled`, but I don't want to go overboard here for no reason.

Reply:
I'd like to keep it simple, but I also see how some deployments may want to limit the number of remote API calls and eg. embedded device with a battery or limited data plan will want to limit sync to once an hour or once a day, so ability to customize the check interval will be requested anyway.

Could we reuse the convention from [`Datastore.GCPeriod`](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#datastoregcperiod)?
It uses the `duration` type (an empty string means the default value).



What if we rename it to `MFSRepinCheckInterval` of type `duration` where empty/missing value simply means `0` (off)? 
That way we keep it simple and give all the flexibility users could want from this feature.

Reply:
I went with the idea to mimic Datastore.GCPeriod. Funnily, polling of a different type is still necessary: to catch changes in the config. Take a look and see what you think.